modules/hub/src/ChannelsService.ts

```diff
@@ -1,56 +1,54 @@
+import { StateGenerator, types, Utils, Validator, big } from './Connext';
 import { redisCache } from './RedisClient'
-import { hasPendingOps } from './vendor/connext/hasPendingOps'
 import log from './util/log'
 import ChannelsDao from './dao/ChannelsDao'
 import Config from './Config'
 import ThreadsDao from './dao/ThreadsDao'
-import { BigNumber } from 'bignumber.js'
-import {
-  channelStateUpdateRowBigNumToString,
-  channelRowBigNumToString,
-  ChannelRow,
-  ChannelStateUpdateRowBigNum,
-} from './domain/Channel'
-import { Validator } from './vendor/connext/validator'
 import ExchangeRateDao from './dao/ExchangeRateDao'
-import { Big, toWeiBigNum } from './util/bigNumber'
-import { ThreadStateUpdateRow } from './domain/Thread'
 import { RedisClient } from './RedisClient'
 import { ChannelManager } from './ChannelManager'
-import {
-  ChannelStateUpdate,
-  UnsignedChannelState,
-  DepositArgs,
-  WithdrawalArgs,
-  ExchangeArgs,
-  convertChannelState,
-  convertThreadState,
-  ThreadState,
-  convertPayment,
-  PaymentArgs,
-  UpdateRequestBigNumber,
-  UpdateRequest,
-  ChannelStateBigNumber,
-  SyncResult,
-  WithdrawalParametersBigNumber,
-  ChannelStateBN,
-  InvalidationArgs,
-  Sync,
-  convertWithdrawalParameters,
-  convertWithdrawal,
-  convertArgs,
-} from './vendor/connext/types'
 import { prettySafeJson, Omit, maybe } from './util'
 import { OnchainTransactionService } from './OnchainTransactionService';
 import DBEngine from './DBEngine';
 import ThreadsService from './ThreadsService';
 import { SignerService } from './SignerService';
 import { OnchainTransactionRow } from './domain/OnchainTransaction';
 import ChannelDisputesDao from './dao/ChannelDisputesDao';
-import { StateGenerator } from './vendor/connext/StateGenerator';
 import { CoinPaymentsDao } from './coinpayments/CoinPaymentsDao'
 import { OnchainTransactionsDao } from './dao/OnchainTransactionsDao';
-
+import { BigNumber as BN } from 'ethers/utils'
+import { assetToWei } from 'connext/dist/lib/bn';
```

Comment:
I'd really really like us to expose everything we need from connext via it's index. Importing individual files will lock up our file structure and make it more likely that future refactors break stuff unexpectedly.
We need to spend some time finalizing the connext API specs. We need to decide & make explicit which utilities are important to expose & which are internal libs that apps shouldn't need to mess with. If assetToWei is useful to apps that integrate connext, it should be available via the Utils or similar.

Reply:
Why are we importing from `dist/` not `index/`? This is really strange to do.

Reply:
Weird, seems to be an issue with the autoimports in my code editor. Fixed it here, and in a couple other files as well.

The reason I changed the export structure was because I added a `hubAddress` to the `Utils` class. This allows the recover functions within the `Utils` class to have the same API as the `Validator` class.

We can change it back to a static class, and export everything through there, or we can change the exports to export a better `utils` object that includes everything. But, I do think we should rename the `Utils` class to `StateUtils` or something